tasks/variable-setup.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Set datadog_checks variable
+  set_fact:
+    agent_datadog_checks: "{{ datadog_checks | default({}, true) }}"
+
+- name: Set datadog_agent_major_version variable
```

Comment:
Do we need this task here? We do exactly the same in `set-parse-version.yml` where it is sufficient AFAICS.